### ISSUE TITLE
Cannot pass arguments to tokio::test

### ISSUE THREAD

**Issue by matthiasbeyer** [d-e-s-o/test-log#21]
Hi,

I have an issue where I need to pass `flavor = "multi_thread", worker_threads = 1` to `tokio::test` like so:

```
#[test_log::test(tokio::test(flavor = "multi_thread", worker_threads = 1))]
async fn my_test() {
    assert!(true);
}
```

But this does not work with this crate, the macro panics.

**Comment by d-e-s-o** [comment]
Should be fixed with version `0.2.9`, which was just released. Thanks for the report!

**Comment by d-e-s-o** [comment]
> Should be fixed with version `0.2.9`, which was just released. Thanks for the report!

Actually, I had to yank this release for now, because of https://github.com/d-e-s-o/test-log/issues/22. You can manually pull in the git repository if you need the functionality immediately.

**Comment by d-e-s-o** [comment]
> Actually, I had to yank this release for now, because of #22. You can manually pull in the git repository if you need the functionality immediately.

Should hopefully be final now with `0.2.10`.
